src/libraries/System.Private.CoreLib/src/System/IO/FileStream.Unix.cs

```diff
@@ -333,7 +334,9 @@ private void FlushOSBuffer()
 
         private void FlushWriteBufferForWriteByte()
         {
+#pragma warning disable CA1416 // Validate platform compatibility, need proper handling
```

Comment:
Please create an issue for this and link it from here. We have to fix it as the code does not work as-is

Reply:
Created https://github.com/dotnet/runtime/issues/44542